app/Notifications/CheckinAccessoryNotification.php

```diff
@@ -35,7 +37,10 @@ public function __construct(Accessory $accessory, $checkedOutTo, User $checkedIn
      */
     public function via()
     {
-        \Log::debug('via called');
+        if (Setting::getSettings()->webhook_selected == 'microsoft'){
+
+             return [MicrosoftTeamsChannel::class];
```

Comment:
We don't want to return here because it prevents emails from being sent (since they are no longer added to `$notifyBy[]` below). I think using `$notifyBy[] = MicrosoftTeamsChannel::class;` like in other places would fix it.